tests/resources/db/latest_schema.sql

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
Autogenerated by tests/store/dump_schema.py, used to validate database schema e.g. after running migrations in tests.